client/package.json

```diff
@@ -34,6 +34,8 @@
     ]
   },
   "dependencies": {
+    "graphql-tag": "^2.0.0",
+    "@babel/core": "^7.0.0",
```

Comment:
can you please pin the added dependencies ?

Reply:
:heavy_check_mark: Dependencies pinned in commit: https://github.com/ledokku/ledokku/pull/421/commits/5c5bc3290d2a4f04a50e4f809e741b4dde014d1e